src/main/react/src/containers/TheHeaderDropdownMssg.js

```diff
@@ -0,0 +1,104 @@
+import React from "react";
+import { CBadge, CDropdown, CDropdownItem, CDropdownMenu, CDropdownToggle, CImg } from "@coreui/react";
+import CIcon from "@coreui/icons-react";
+
+const TheHeaderDropdownMssg = () => {
+  const itemsCount = 4;
+  return (
+    <CDropdown inNav className="c-header-nav-item mx-2" direction="down">
+      <CDropdownToggle className="c-header-nav-link" caret={false}>
+        <CIcon name="cil-envelope-open" />
+        <CBadge shape="pill" color="info">
+          {itemsCount}
+        </CBadge>
+      </CDropdownToggle>
+      <CDropdownMenu className="pt-0" placement="bottom-end">
+        <CDropdownItem header tag="div" color="light">
+          <strong>You have {itemsCount} messages</strong>
+        </CDropdownItem>
+        <CDropdownItem href="#">
+          <div className="message">
+            <div className="pt-3 mr-3 float-left">
+              <div className="c-avatar">
+                <CImg src={"avatars/7.jpg"} className="c-avatar-img" alt="admin@bootstrapmaster.com" />
```

Comment:
why are those things hard-coded? Is this more meant as an example?

Reply:
I think so.